src/modules/playground.ts

```diff
@@ -76,6 +88,30 @@ export class PlaygroundModule extends Module {
 		}
 	}
 
+	@listener({ event: 'message' })
+	async onPGLinkAttachment(msg: Message) {
+		const attachment = msg.attachments.find(a => a.name === 'message.txt');
+		if (!attachment) return;
+		const content = await fetch(attachment.url).then(r => r.text());
+		await this.onLongPGLink(msg, content, !!msg.content, true);
```

Comment:
Overall, rather than calling the other handler here, with overloads, I think I'd prefer to create a helper function which is called by both.

Reply:
I'm up to play around with this refactoring and send a PR to your PR :P

Reply:
https://github.com/tjjfvi/community-bot/pull/1